snuba/query/processors/prewhere.py

```diff
@@ -1,8 +1,8 @@
 from typing import Optional, Sequence
 
 from snuba import settings, util
-from snuba.query.query import Query
-from snuba.query.query_processor import QueryProcessor
+from snuba.clickhouse.processors import QueryProcessor
+from snuba.clickhouse.query import Query
```

Comment:
(Ok, this one also probably belongs in `snuba.clickhouse` too.)

Reply:
Does it? I would have put together with the storages (datasets/storages)

Reply:
I think that the physical storage implementations probably belong in `snuba.clickhouse` too, so I don't think these are incompatible views. This also reflects my understanding of  https://github.com/getsentry/snuba/pull/880#discussion_r415955384, specifically this part:

> storage will have a generic interface on top with the same parameter and all our storages will be clickhouse specific and extend from this

Reply:
yeah, moving all the clickhouse storages into snuba/clickhouse makes sense.
Will see what I can move in this PR without loosing git history.

Reply:
I think it would be fine (better, really) to do this as a follow up change (as noted with https://github.com/getsentry/snuba/pull/880#discussion_r416777412) since this is one is pretty large as-is.